CHANGELOG.md

```diff
@@ -11,23 +11,23 @@ We are pleased to announce the latest CKEditor 5 release, focusing on performanc
 
 This release introduces another set of performance related improvements, focused on faster editor initialization for huge documents. The initialization time was lowered by further 15% to 45%, depending on the tested sample.
 
-The combined improvements introduced in recent releases amount to around 65%-80% lower loading time in total, which means 3-5x higher editor loading speed. As the gain is not linear, bigger documents see even better improvement (+10x speed).
+The combined improvements introduced in recent releases amount to around 65%-80% lower loading time in total, which means the editor will load 3-5x faster. As the gain is not linear, bigger documents see even better improvement (more than 10x faster).
 
 Moreover, all these improvements positively impact document save time (`editor.getData()`), which should help with autosave issues, among others.
 
 We still actively work in this area, so you may expect even more editor load and save efficiency improvements in the upcoming releases.
 
-#### 🔨**Bug Fixes and improvements**
+#### 🔨 **Bug Fixes and improvements**
 
 * **Comments enhancements**:
     * **Data export options**: We introduced the `showCommentHighlights` option in `editor.getData()`, that changes the comment marker conversion, allowing for styling comments in the output. Perfect for showing what was commented in [Export to PDF](https://ckeditor.com/docs/ckeditor5/latest/features/converters/export-pdf.html), for example.
     * **Inline mode improvements**: We addressed a problem where comment annotations in inline mode did not close properly when clicking elsewhere in the content.
     * **Thread management**: We resolved an issue where creating a new thread was not interrupted when the corresponding marker was removed from the content, ensuring better stability during collaborative editing.
 * **Revision History update**:
-    * **Restore functionality**: We disabled the ability to restore the current (edited and not saved) revision, as it was illogical and led to some non-obvious behaviors.
+    * **Restore functionality**: We disabled the ability to restore the current (edited, not saved) revision, as it represents current content, so there is nothing to restore. At the same time, using it led to some non-obvious behaviors.
 * **Image handling**: We resolved an issue where images in the uploading state could be deleted when dragged and dropped within the editor. Keep dragging, even when it is not there 🙈.
 
-### 🎄
+### 🎄 **Happy holidays!**
```

Comment:
This looked like some random out of place emoji.

Reply:
How about we stop using emojis in technical docs at all?

Reply:
As we need this changelog, I'd like to discuss your proposal Bartek.